### PR TITLE
Man page for sumaxy converted to .rst

### PR DESCRIPTION
**Pull request by ostruzina**
Man page for sumaxy has been converted to .rst.

Currently working on the makefile to make sure that the man page is generated during build.
This is work in progress, please do not merge yet.

Related issue #6

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Roman Neuhauser <rneuhauser+sumaxy@sigpipe.cz>;
```
Comment (rwmanos) [inline AUTHORS]: Good job! There is a minor issue which isn't clearly visible.
The authors list must be of the form

```
:Authors:
 Author 1
 Author 2
```
With a leading whitespace in front of every author.

For repose I inserted that whitespace manually in front of every name in the AUTHORS file. Feel free to do the same or to come up with a better way :)

Comment (ostruzina) [reply AUTHORS]: Thanks, @rwmanos! I changed the authors file accordingly and added you in.

File: GNUmakefile.in [hunk 2]
```diff
@@ -48,14 +56,14 @@ check: $(tests) | all
 
 .PHONY: clean
 clean:
-	rm -f $(commands) $(testdir)/*.err
+	rm -f $(commands) $(testdir)/*.err $(addprefix $(man1dir)/,$(man1pages)) $(addsuffix .rst,$(addprefix $(man1dir)/,$(man1pages)))
```
Comment (rwmanos) [inline GNUmakefile.in:59]: The following seems to delete the installed files 
```
make clean
rm -f sumaxy tests/*.err /usr/local/share/man/man1/sumaxy.1 /usr/local/share/man/man1/sumaxy.1.rst
```
while `make clean` should clean the tmp/build directory. You might want to use this code for `make uninstall` and try the following for `make clean`
```
rm -f $(commands) $(testdir)/*.err $(manpages) $(addsuffix .rst,$(manpages))
```
@mimi1vx you were a bit fast.

Comment (ostruzina) [reply GNUmakefile.in:59]: I see - thank you!

**Pull request merged**